src/HorizontalImageList/index.js

```diff
@@ -93,6 +97,21 @@ class HorizontalImageList extends Component {
     }
 
     const { imageSize, imageRounding, shape, shadow } = imageChild
+
+    if (
+      !imageList ||
+      typeof navigator.userAgent === undefined ||
+      (!imageList[0] && !listEmptyState)
+    ) {
+      if (hasUpdatedLoadingStates) {
+        return (
+          <View style={{ height: imageSize, justifyContent: 'center' }}>
+            <ActivityIndicator />
+          </View>
+        )
+      } else return <View style={{ height: imageSize }}></View>
```

Comment:
```suggestion
      if (hasUpdatedLoadingStates) {
        return (
          <View style={{ height: imageSize, justifyContent: 'center' }}>
            <ActivityIndicator />
          </View>
        )
      } else {
        return <View style={{ height: imageSize }}></View>
      }
```

The `else` block is for code-style consistency with your other conditionals in this PR. You can also just drop the `else` block entirely, since it's unnecessary. I don't have a preference, but if you're going to make that change, please make it everywhere.